{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/taskapp/celery.py

```diff
@@ -27,25 +27,26 @@ def ready(self):
         app.autodiscover_tasks(lambda: installed_apps, force=True)
 
         {% if cookiecutter.use_sentry == 'y' -%}
-        if hasattr(settings, 'RAVEN_CONFIG'):
+        if hasattr(settings, "SENTRY_DSN"):
             # Celery signal registration
-{% if cookiecutter.use_pycharm == 'y' -%}
-	    # Since raven is required in production only,
+            {% if cookiecutter.use_pycharm == 'y' -%}
+	        # Since Sentry is required in production only,
             # imports might (most surely will) be wiped out
             # during PyCharm code clean up started
             # in other environments.
             # @formatter:off
-{%- endif %}
-            from raven import Client as RavenClient
-            from raven.contrib.celery import register_signal as raven_register_signal
-            from raven.contrib.celery import register_logger_signal as raven_register_logger_signal
-{% if cookiecutter.use_pycharm == 'y' -%}
+            {%- endif %}
+            import sentry_sdk
+            from sentry_sdk.integrations.celery import CeleryIntegration
+            from sentry_sdk.integrations.logging import LoggingIntegration
+            {% if cookiecutter.use_pycharm == 'y' -%}
             # @formatter:on
-{%- endif %}
-
-            raven_client = RavenClient(dsn=settings.RAVEN_CONFIG['dsn'])
-            raven_register_logger_signal(raven_client)
-            raven_register_signal(raven_client)
+            {%- endif %}
+            sentry_logging = LoggingIntegration(
```

Comment:
When executed like this, it appears to create a conflict in Sentry an silence all errors from loggin at all. To me it seems to be that the initialization must be only once, if Celery and Sentry are both configured, the `init` script is executed two times. My suggestion is to remove 
```
sentry_logging = LoggingIntegration(
    level=settings.SENTRY_LOG_LEVEL,  # Capture info and above as breadcrumbs
    event_level=None,  # Send no events from log messages
)
sentry_sdk.init(dsn=settings.SENTRY_DSN, integrations=[sentry_logging, CeleryIntegration()])
```
from here and in `production.py` do something like:
```
{% if cookiecutter.use_celery == 'y' -%}
sentry_sdk.init(
    SENTRY_DSN,
    integrations=[sentry_logging, DjangoIntegration(), CeleryIntegration()]
)
{% else %}
sentry_sdk.init(
    SENTRY_DSN,
    integrations=[sentry_logging, DjangoIntegration()]
)
{%- endif %}
```

Reply:
@elyak123 I can't thank you enough for this comment.

Reply:
@dekoza do you mean that you had the same problem and this fixed it?

Reply:
@browniebroke My project was not reporting errors except for errors in the settings.py. I even made a special view raising excpetion to test if sentry works. I lost several hours trying to debug this and this comment saved me from putting even more hours to this.